public/javascripts/now.js

```diff
@@ -156,7 +158,7 @@ function processKey(event) {
         toggleCookie('extendedOn');
         showE = undefined;
       }
-      toggleDisplay('#userLine', showE);
+      toggleDisplay(resources.state.extendedInfo, showE);
```

Comment:
If you add a new hotkey `t`, can you add something like this to the top of the `processKey` function to make sure it doesn't get triggered when doing something like `ctrl`+`t`?

```javascript
// Return if modifier key is held down
if (window.event.ctrlKey || window.event.metaKey)
  return;
```